server/src/main/java/com/broadcom/lsp/cobol/core/visitor/VariableDefinitionDelegate.java

```diff
@@ -238,11 +268,15 @@ void switchSection(@NonNull DataDivisionSection section) {
   @NonNull
   ResultWithErrors<Collection<Variable>> finishDefinitionAnalysis() {
     switchSection(NOT_INITIALIZED);
-    return new ResultWithErrors<>(variables, errors);
+    return new ResultWithErrors<>(new ArrayList<>(variables), new ArrayList<>(errors));
```

Comment:
Do we edit these lists later? At least errors are not, why can't be List.of(variables) ?

Reply:
`List.of()` will return a list of list in this case